helpers/event/event_slack_notifier.py

```diff
@@ -0,0 +1,11 @@
+import requests
+import celery
+from config import Config
+
+"""
+This function will call slack bot url
+"""
+
+@celery.task(name="notify-slack")
+def notify(event_Id):
```

Comment:
Please make this function name more descriptive. `notify` what? Think of a better function name. Ask yourself what exactly the function is doing then name it appropriately.